### PR TITLE
Maui 874730 Added UG documentation for SfRotator control in .NET MAUI

### PR DESCRIPTION
**Pull request by HariharanChokkalingam**
Hi Team,

I added Rotator User Guide content to this repository

Regards,
Hariharan C

### CODE REVIEW

File: MAUI/Rotator/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,574 @@
...
+
+	using Syncfusion.Maui.Rotator;
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Create a Simple SfRotator 
+
+The [`SfRotator`](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfRotator.XForms.SfRotator.html) control is configured entirely in C# code or by using XAML markup. The following steps explain on how to create a [`SfRotator`](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfRotator.XForms.SfRotator.html) and configure its elements,
```
Comment (anandhan-rajagopal) [inline MAUI/Rotator/getting-started.md:75]: Remove all the old API reference

Comment (HariharanChokkalingam) [reply MAUI/Rotator/getting-started.md]: Yes Sir, Now Old API references have been removed.

File: MAUI/Rotator/Adding-Looping-and-Delays.md [hunk 2]
```diff
@@ -0,0 +1,376 @@
+---
+layout: post
+title: Adding Looping and Delays in .NET MAUI Rotator control | Syncfusion
+description: Learn here all about Adding Looping and Delays support in Syncfusion .NET MAUI Rotator (SfRotator) control and more.
```
[the remark this answers is not in the document]
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Adding-Looping-and-Delays.md]: I had addressed the Proposed concerns.

**Comment by HariharanChokkalingam** [comment]
HI @Chrispineagunja ,

 I have addressed the Proposed concerns.

File: MAUI/Rotator/Adding-Looping-and-Delays.md [hunk 3]
```diff
@@ -0,0 +1,376 @@
...
+title: Adding Looping and Delays in .NET MAUI Rotator control | Syncfusion
+description: Learn about Adding Looping and Delay support in Syncfusion .NET MAUI Rotator (SfRotator) control and more.
+platform: maui 
+control: Rotator
+documentation: ug
+---
+
+# Adding Looping and Delays in .NET MAUI Rotator (SfRotator)
+
+Looping and delay can be enabled in the `SfRotator` control, and you can customize the Text and Navigation direction.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Adding-Looping-and-Delays.md:12]: Recheck this sentence

File: MAUI/Rotator/Dots-Customization.md [hunk 4]
```diff
@@ -0,0 +1,291 @@
...
+documentation: ug
+---
+
+# Dots Customization in .NET MAUI Rotator (SfRotator)
+
+The Rotator control supports customizing dots' border color, selected dots' color, and unselected dots' color.
+
+## DotsBorder Color
+
+The `DotsBorderColor` property is used to customize the color of the dots border in the `SfRotator`.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Dots-Customization.md:16]: Use DotsStroke

File: MAUI/Rotator/Dots-Customization.md [hunk 5]
```diff
@@ -0,0 +1,291 @@
...
+    <ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+                xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                xmlns:syncfusion="clr-namespace:Syncfusion.Maui.Rotator;assembly=Syncfusion.Maui.Rotator"
+                xmlns:local="clr-namespace:Rotator"
+                x:Class="Rotator.Rotator">
+        <ContentPage.BindingContext>
+            <local:RotatorViewModel/>
+        </ContentPage.BindingContext>
+        <ContentPage.Content>
+            <syncfusion:SfRotator x:Name="rotator"
```
Comment (Choza-rajan) [inline MAUI/Rotator/Dots-Customization.md:31]: Is this code deployed in MAUI?

File: MAUI/Rotator/Dots-Customization.md [hunk 6]
```diff
@@ -0,0 +1,291 @@
...
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+![SfRotator with dots border color](images/DotsBorderColor.png)
+
+## Selected Dot Color
+
+The `SelectedDotColor` property is used to customize the color of selected dots in the `SfRotator`.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Dots-Customization.md:107]: The `SelectedDotColor` property is used to customize the color of selected dot in the `SfRotator`.

File: MAUI/Rotator/Events.md [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+platform: maui 
+control: Rotator 
+documentation: ug
+---
+
+# Events in .NET MAUI Rotator (SfRotator)
+
+## Selection Changed
+
+The SelectionChanged event is used to notify when the selection is changed by swiping or dynamically setting the `SelectedIndex` property of the `SfRotator`.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Events.md:14]: Here use SelectedIndexChanged event

File: MAUI/Rotator/Getting-started.md [hunk 8]
```diff
@@ -0,0 +1,574 @@
...
+
+{% endhighlight %}
+{% endtabs %}
+
+
+## Create a Simple SfRotator 
+
+The `SfRotator` control is configured entirely in C# code or by using XAML markup. The following steps explain on how to create a `SfRotator` and configure its elements,
+
+* Adding namespace for the added assemblies.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Getting-started.md:77]: 2nd time explaining about namespace. Is this needed?

File: MAUI/Rotator/Getting-started.md [hunk 9]
```diff
@@ -0,0 +1,574 @@
...
+
+	rotator.BindingContext = new RotatorViewModel();
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![RotatorImages](images/RotatorImages.png)
+
+I> Rotator's Images are placed within the application folder for Android, iOS and UWP with build action Android Resource, Bundled Resource and Content respectively.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Getting-started.md:380]: recheck this sentence

File: MAUI/Rotator/Loading-Online-Images.md [hunk 10]
```diff
@@ -0,0 +1,142 @@
...
+title: Loading Online Images in .NET MAUI Rotator control | Syncfusion
+description: Learn about Loading Online Images support in Syncfusion .NET MAUI Rotator (SfRotator) control and more.
+platform: maui 
+control: Rotator 
+documentation: ug
+---
+
+# Loading Online Images in .NET MAUI Rotator (SfRotator)
+
+This section describes loading the online images in the `SfRotator` Control.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Loading-Online-Images.md:12]: Once recheck this code with all 4 platforms

File: MAUI/Rotator/Migration.md [hunk 11]
```diff
@@ -0,0 +1,38 @@
...
+
+## Properties
+
+<table> 
+<tr>
+<th>Xamarin SfRotator</th>
+<th>.NET MAUI SfRotator</th>
+<th>Description</th></tr>
+<tr>
+<td>{{'[DotsBorderColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.Buttons.ISwitchSettings.html#Syncfusion_XForms_Buttons_ISwitchSettings_TrackBorderWidth)'| markdownify }}</td>
```
Comment (Choza-rajan) [inline MAUI/Rotator/Migration.md:33]: recheck the link

File: MAUI/Rotator/Navigation-Modes.md [hunk 12]
```diff
@@ -0,0 +1,291 @@
...
+control: Rotator
+documentation: ug
+
+---
+
+# Navigation Modes in .NET MAUI Rotator (SfRotator)
+
+The `NavigationStripMode` property specifies the appearance of navigation bar items. The image data can be selected either by `Thumbnail` or by `Dots`navigation modes.
+
+* `Thumbnail` - The slider items will also be loaded in the thumbnail view. When a thumbnail item is clicked, the slider will switch to the corresponding image data.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Navigation-Modes.md:16]: use it like sub header

File: MAUI/Rotator/Navigation-Modes.md [hunk 13]
```diff
@@ -0,0 +1,291 @@
...
+        }
+    }	
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Thumbnail Mode Image](images/NavigationStripMode.png)
+
+* `Dots` - The slider items will also be loaded in dots view. When a dot item is clicked, the slider will switch to the corresponding image data.
```
Comment (Choza-rajan) [inline MAUI/Rotator/Navigation-Modes.md:105]: use it like sub header

File: MAUI/Rotator/Getting-started.md [hunk 14]
```diff
@@ -0,0 +1,574 @@
...
+    }
+        
+{% endhighlight %}
+
+{% endtabs %}
+
+
+## Add Rotator Items
+
+We can populate the rotator's items by using any one of the following ways,
```
Comment (Choza-rajan) [inline MAUI/Rotator/Getting-started.md:139]: In getting started only use RotatorItem. In **populating items topic** covers the item template and item content topics

File: MAUI/Rotator/Placement-Modes.md [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+platform: maui 
+control: Rotator 
+documentation: ug
+---
+
+# Placement Modes in .NET MAUI Rotator (SfRotator)
+
+By default, the rotator control displays the dots of each rotator item. It can be changed to any of the following types:
+
+* `Default`
```
Comment (Choza-rajan) [inline MAUI/Rotator/Placement-Modes.md:14]: explain this 3 topics

File: MAUI/Rotator/Sliding-Direction.md [hunk 16]
```diff
@@ -0,0 +1,110 @@
...
+            {
+                get { return _image; }
+                set { _image = value; }
+            }
+        }
+    }
+
+{% endhighlight %}
+
+{% endtabs %}
```
Comment (Choza-rajan) [inline MAUI/Rotator/Sliding-Direction.md:108]: Once recheck all images and use common size for all images.

File: MAUI/Rotator/Adding-Looping-and-Delays.md (hunk 3 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Adding-Looping-and-Delays.md:12]: I had changed like Looping and delay can be enabled in the `SfRotator` control in .NET MAUI.

File: MAUI/Rotator/Dots-Customization.md (hunk 4 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Dots-Customization.md:16]: changed to DotsBorderColor  to  DotsStroke

File: MAUI/Rotator/Dots-Customization.md (hunk 5 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Dots-Customization.md:31]: Yes Sir, I have run all the code snippet used.

File: MAUI/Rotator/Dots-Customization.md (hunk 6 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Dots-Customization.md:107]: changed dots to dot

File: MAUI/Rotator/Getting-started.md (hunk 8 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Getting-started.md:77]: I had remove the namespace explaination in Create a Simple SfRotator

File: MAUI/Rotator/Migration.md (hunk 11 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Migration.md:33]: changed with correct link

File: MAUI/Rotator/Getting-started.md (hunk 9 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Getting-started.md:380]: I had remove that notes.

File: MAUI/Rotator/Navigation-Modes.md (hunk 12 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Navigation-Modes.md:16]: used as subheading

File: MAUI/Rotator/Navigation-Modes.md (hunk 13 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Navigation-Modes.md:105]: used as subheading

File: MAUI/Rotator/Sliding-Direction.md (hunk 16 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Sliding-Direction.md:108]: resize the Images

File: MAUI/Rotator/Placement-Modes.md (hunk 15 above)
Comment (HariharanChokkalingam) [reply MAUI/Rotator/Placement-Modes.md]: Added content for 3 topics

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**